src/add-api/index.md

```diff
@@ -0,0 +1,31 @@
+# How do I add scoreboards and trophies to my game?
+
+First, you need to implement the Game API in your game's code. Then, you can add scoreboards and trophies to your game.
+
+## Scoreboards
+
+You can add multiple scoreboards to a game, so you could have, for example, one for each level of the game, as well as a global scoreboard. Or, you could make one that ranks players by something besides points, like total time or number of jumps. You have control over sorting options, so you could make a traditional scoreboard, where the same person can occupy multiple slots, or you could make it so that only someone's best score is displayed. You can enable guest scoring, so players don't need an account to submit a score. You can even attach extra, hidden data to scores (for verification, tracking, or other uses).
```

Comment:
YE  needs to rewrite. Break into multiple paragraphs. Add an intro something like "Scoreboards are a great way to get gamers to engage with your game by competing."

- [ ] Paragraph split out by functionality. Descending and Ascending scores.